zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -858,47 +858,53 @@ mod test {
         }
         caller_bytecode.append(&bytecode! {
             PUSH32(Address::repeat_byte(0xff).to_word())
-            PUSH2(10000)
+            // gas 10032 for static call
+            PUSH2(10000 + 32*(!is_call as u64))  // 10000 for call
             .write_op(opcode)
             STOP
         });
+        let (jump_dest, gas) = if is_call { (43, 132) } else { (41, 129) };
         // The following bytecode calls itself recursively if gas_left is greater than
         // 100, and halts with REVERT if gas_left is odd, otherwise just halts
         // with STOP.
         let mut callee_bytecode = bytecode! {
             GAS
             PUSH1(100)
             GT
-            PUSH1(43)
+            PUSH1(jump_dest) // jump dest
```

Comment:
what do you think we expand directly here?
```suggestion
            PUSH1(if is_call {43} else {41}) // jump dest
```